test/tests/testService.test.ts

```diff
@@ -282,6 +282,7 @@ describe('Apex Test Suites', async () => {
         'TestClass1.method1,namespace.TestClass2.method1,TestClass2.method2';
 
       const testService = new TestService(mockConnection);
+      // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
Is this comment meant to be info or was this a debugging line that should be removed?